matrix_procedures/display_aequilibrae_formats_dialog.py

```diff
@@ -40,31 +52,51 @@ def __init__(self, iface):
 
         self.error = None
 
+        formats = ["Aequilibrae matrix(*.aem)", "Aequilibrae dataset(*.aed)"]
+        if has_omx:
+            formats.insert(1, "Open Matrix(*.omx)")
         self.error = None
         self.data_path, self.data_type = GetOutputFileName(
             self,
             "AequilibraE custom formats",
-            ["Aequilibrae dataset(*.aed)", "Aequilibrae matrix(*.aem)"],
-            ".aed",
+            formats,
+            ".aem",
             standard_path(),
         )
 
         if self.data_type is None:
             self.error = "Path provided is not a valid dataset"
             self.exit_with_error()
-
-        self.data_type = self.data_type.upper()
-
-        if self.data_type == "AED":
-            self.data_to_show = AequilibraEData()
-        elif self.data_type == "AEM":
+        else:
+            self.data_type = self.data_type.upper()
+
+        if self.data_type in ["AED", "AEM"]:
```

Comment:
```suggestion
        if self.data_type in ["AED", "AEM"]:
            load_aequilibrae_formatted_data()
        elif self.data_type == "OMX":
            load_omx_format_data()
```

You can also put in the stuff about computation_views in those methods

Reply:
I need to load the matrix first...